src/lib/input_schema.js

```diff
@@ -65,6 +71,36 @@ const readInputSchema = async (forcePath) => {
     };
 };
 
+/**
+ * Goes to the Actor directory and creates INPUT.json file from the input schema prefills.
+ * @param {string} actorFolderDir
+ */
+const createPrefilledInputFileFromInputSchema = async (actorFolderDir) => {
+    const currentDir = process.cwd();
+    try {
+        process.chdir(actorFolderDir);
+        const { inputSchema } = await readInputSchema();
+        if (inputSchema) {
+            const validator = new Ajv({ strict: false });
```

Comment:
Let's put note somewhere that the code is same as in console and we should move it to shared place.